net.sourceforge.quakespasm.Quakespasm.yml

```diff
@@ -0,0 +1,68 @@
+app-id: net.sourceforge.quakespasm.Quakespasm
+runtime: org.freedesktop.Platform
+runtime-version: '23.08'
+sdk: org.freedesktop.Sdk
+command: quakespasm.sh
+
+finish-args:
+  # hardware 3D
+  - --device=dri
+  # X11 + XShm access
+  - --share=ipc
+  - --socket=x11
+  # Audio
+  - --socket=pulseaudio
+
+cleanup:
+  - /include
+  - "*.a"
+  - "*.la"
+  - /lib/pkgconfig
+
+modules:
+
+  - shared-modules/libmad/libmad.json
+
+  - name: libxmp
+    buildsystem: autotools
+    sources:
+      - type: git
+        url: https://github.com/libxmp/libxmp/
+        tag: libxmp-4.6.0
+        commit: 8201d26cf933688a8be64292457c429fd8e654ab
+        x-checker-data:
+          type: git
+          tag-pattern: ^libxmp-(.*)$
+
+  - name: quakespasm
+    sources:
+      - type: git
+        url: https://github.com/sezero/quakespasm
+        tag: quakespasm-0.96.1
+        commit: 09e086ba955d179e58aaf5c74e256dcdf9dcdad4
```

Comment:
This tag is missing the appdata https://github.com/sezero/quakespasm/tree/quakespasm-0.96.1 ?

Reply:
You can remove the tag and use the commit only to specify the recent commit sha or ask the maintainer to create a new tag.